src/models/api/CollectionItemResponse.ts

```diff
@@ -0,0 +1,16 @@
+export type CollectionItemResponse = {
+  description: {
+    summary: string,
+    returns: string,
+    examples: string[],
+  },
```

Comment:
Curious why here we provide an entire object, but in response, we map and receive only `description.summary`?

Reply:
Oh I see, because it's broken down in the main model. But why can't we use object there too?

Reply:
we could, I just felt like a flatter model was more typical for us